### PR TITLE
GH1 - xts allocator server

### PR DESCRIPTION
**Pull request by zghp**

### CODE REVIEW

File: logo.png
[the remark this answers is not in the document]
Comment (zghp) [reply logo.png:1]: I optimised the image. It's generated using ai

[the remark this answers is not in the document]
Comment (mhughesacn) [reply logo.png:1]: The size is fine now - thank you.  But I'm not sure about the origin.  Do you have a permission for AI-generated imagery that I could refer to?

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <img src="/logo.png" width="150" style="border-radius: 50%;"/>
+        </br>
+        <h1>Welcome to the XTS Allocator Server</h1>
+        <p>This is the main page for managing allocations and devices.</p>
+        <a href="/list_slots">View Slots</a>
```
Comment (TB-1993) [inline templates/index.html:30]: This is a very neat idea, nicely done 😄

**Review by TB-1993 — CHANGES_REQUESTED** [review]
This looks great, just need to fill out the requirements.txt. The next step is writing a build script to build it into a single binary. But we'll do that as the next ticket.

**Review by TB-1993 — APPROVED** [review]
Looks good, ship it. Can you please raise another ticket on this repo to create an installation and build script? That will integrate it with pyinstaller for building it as a single binary for people to run.

**Pull request merged**